source/change-streams/tests/change-streams-errors.yml

```diff
@@ -70,6 +70,18 @@ tests:
         arguments:
           document:
             z: 3
+    expectations:
```

Comment:
If I understand [SPEC-1351](https://jira.mongodb.org/browse/SPEC-1351) correctly, the concern is that a driver might resume and produce more than a single `command_started_event`? If so, I don't think adding this expectation achieves your goal, as `expectations` is permitted to be a subset of the total number of commands observed by APM. This is why I opened [SPEC-1341](https://jira.mongodb.org/browse/SPEC-1341) for another test.